### ISSUE TITLE
Version 0.15.0 didn't get published

### ISSUE THREAD

**Issue by sideeffffect** [circe/circe-refined#4]
Only 0.15.0-M1 is on Maven Central
 * https://repo1.maven.org/maven2/io/circe/circe-refined_2.13/
 * https://github.com/circe/circe-refined/actions/runs/10613572612/job/29417749292#step:28:228

There were warnings like this in the task `tlCiRelease`, they may be relevant. But the build is still marked as green /cc @armanbilge
```
[2/243] Execution failed: Failed to parse the response from the server: Response([200: OK],Vary:Accept-Charset, Accept-Encoding, Accept-Language, Accept,Server:Nexus/2.15.1-02 Noelios-Restlet-Engine/1.1.6-SONATYPE-5348-V8,X-Frame-Options:SAMEORIGIN,Connection:keep-alive,Content-Length:428,X-Content-Type-Options:nosniff,Content-Type:application/json;charset=utf-8,Date:Thu, 29 Aug 2024 10:51:24 GMT): Failed to read the input msgpack data with codec: wvlet.airframe.codec.CollectionCodec$SeqCodec@1e67584. Retrying in 4.50 sec.  - (SonatypeClient.scala:146)
```
/cc @hamnis

**Comment by hamnis** [comment]
circe-refined 0.15.0 was unfortunately published with the wrong coordinates.
see #3

**Comment by hamnis** [comment]
Fixed in 0.15.1 which was just released. Should be available in maven central now.

**Comment by mzuehlke** [comment]
It's available: https://repo1.maven.org/maven2/io/circe/circe-refined_2.13/0.15.1/ 
Thanks
